sdk/analytics/src/test/java/com/klaviyo/analytics/KlaviyoTest.kt

```diff
@@ -94,6 +95,10 @@ internal class KlaviyoTest : BaseTest() {
     @After
     override fun cleanup() {
         Registry.get<State>().reset()
+        Registry.unregister<Config>()
```

Comment:
This is the key part for the failing test on Ajay's PR.

Reply:
Can you tell me more on how this was causing the issue. I would have thought it's related to state.

Reply:
Oh sorry, it is state, commented on the wrong line

Reply:
The issue was that in the `setup` of this test class we call `Klaviyo.initialize` and within initialize we register config, state, state side effects, so in teardown we have to unregister those. 
My idea for a bigger fix would be to isolate registry a bit more so that each test starts with a clean registry and you don't have to unregister individually like this